rules/sinks/third_parties/sdk/openai/java.yaml

```diff
@@ -0,0 +1,14 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+  
+  - id: ThirdParties.SDK.OpenAI
+    name: OpenAI
+    domains:
+      - "openai.com"
+    patterns:
+      - "(?i).*(OpenAiService|OpenAiStreamClient|OpenAiClient|OpenAiApi).*"
```

Comment:
This rule looks broad, can we include the package as this java

Reply:
Agree. Lets not use such patterns - these would hardly match